.dockerignore

```
@@ -0,0 +1 @@
+Gemfile.lock
```

I'd exclude some of the files which are not required for the local test runs:

```suggestion
Gemfile.lock
*.asciidoc
*.md
docs/
.ci/
```